src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = getSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    public static SessionFactory initSessionFactory() {
+        return new Configuration().configure().buildSessionFactory();
+    }
```

Comment:
This method `initSessionFactory()` is not used anywhere. It's better to remove it. If you want to keep it, you should handle HibernateException here. Also, you need to close SessionFactory in case of initialization failure.